network/basics/uri.py

```diff
@@ -354,6 +386,7 @@ def main():
     argument_spec = url_argument_spec()
     argument_spec.update(dict(
         dest = dict(required=False, default=None, type='path'),
+        source = dict(required=False, default=None, type='path'),
         url_username = dict(required=False, default=None, aliases=['user']),
         url_password = dict(required=False, default=None, aliases=['password']),
         body = dict(required=False, default=None, type='raw'),
```

Comment:
SHould source and body be marked mutually_exclusive here?  You can see an example in the pip module: https://github.com/ansible/ansible-modules-core/blob/devel/packaging/language/pip.py#L378